akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventsourcedReplayingSnapshot.scala

```diff
@@ -108,7 +109,7 @@ private[akka] class EventsourcedReplayingSnapshot[C, E, S](override val setup: E
         becomeReplayingEvents(state, seqNr, toSnr)
 
       case LoadSnapshotFailed(cause) ⇒
-        cancelRecoveryTimer(setup.timers)
+        setup.cancelRecoveryTimer()
```

Comment:
Cancelled inside of `onRecoveryFailure` as well (and that is consistent with the other timeout-cancels), so this one isn't needed.